LondonBritishLibrary/orient/BLorient682.xml

```diff
@@ -0,0 +1,491 @@
+                                <title type="complete" ref="LIT5549TribesApostles"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.23">
+                                <locus from="195r"/>
+                                <title type="complete" ref="LIT5835DatesApostles"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+
+                        </msItem>
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>   
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">195</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.36</height>
+                                        <width>11.5</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="27 29"/>
+                            </layoutDesc>
+
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1730" notAfter="1755"/>
+                                <desc>Written in a good hand.</desc>
+                            </handNote>
+                        </handDesc>
+
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#2r"/>
+                                    <desc type="OwnershipNote">The note states that the manuscript
+                                        belongs to the church of <placeName ref="INS0101MadhaneAlam"
+                                        />.</desc>
+                                    <q xml:lang="gez">ገድለ፡ ሐዋርያት፡ ዘ<placeName
+                                        ref="INS0101MadhaneAlam">መድኃኔ፡ ዓለም።</placeName>
+                                    </q>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                    
+                        
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1">In the original boards, lined with striped cloth and covered with stamped leather.</decoNote>
```

Comment:
stamped or tooled?

Reply:
yes, you are right, I just took Wright's terminology. Since there are digitized manuscripts of his catalogue I could safely confirm that stamped means blind tooled. I will correct the other records in a separate branch.